internal/storage/types.go

```diff
@@ -13,7 +13,7 @@ import (
 type TotalSupply = big.Int
 
 type Paginated interface {
-	models.Transaction | models.Dag | models.Pbft
+	models.Account | models.Transaction | models.Dag | models.Pbft
```

Comment:
You added new generic method for the Account. So no need to add it here as you aren't using it in `GetObjectsPage` or other methods